### DISCUSSION TITLE
Plain objects error

### DISCUSSION THREAD

**Discussion by TuinboonDev** [tinacms/tinacms#4778] [Help]
Hello, I just tried setting up the tina-cloud-starter and converting it to be self hosted but every time I run tina with `pnpm dev` it gives the following error when opening the home page `Error: Only plain objects, and a few built-ins, can be passed to Client Components from Server Components. Classes or null prototypes are not supported.` any way to fix this?

**Comment by kldavis4** [comment]
there's a [thread](https://discord.com/channels/835168149439643678/1235175504441512057/1235175504441512057) on discord discussing the issue and an existing issue: https://github.com/tinacms/tinacms/issues/4517
